src/app/Http/Controllers/AdminController.php

```diff
@@ -42,4 +42,14 @@ public function redirect()
         // The '/admin' route is not to be used as a page, because it breaks the menu's active state.
         return redirect(backpack_url('dashboard'));
     }
+
+    /**
+     * Show the 404 error page.
+     *
+     * @return \Illuminate\View\View
+     */
+    public function notFound()
```

Comment:
```suggestion
    public function backpackHttpNotFoundView()
```

I think this name better reflects what this function is about.